build-scripts/ubuntu-1604/prepare-package.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -xe
```

Comment:
Please use python scripts instead of bash scripts from now on--and convert bash to python as a refactoring task, over time, so that we are better positioned to support builds on Windows.